packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -414,3 +414,51 @@ test.describe('`position`', () => {
         });
     });
 });
+
+test.describe('Prop: `isFooterPinned`', () => {
+    [true, false].forEach((isFooterPinned) => {
+        test(`when isFooterPinned is: ${isFooterPinned}`, async ({ mount, page }) => {
+            await mount(PieModal, {
+                props: {
+                    heading: 'This is a modal heading',
+                    isOpen: true,
+                    isFooterPinned,
+                    leadingAction: {
+                        text: 'Confirm',
+                        variant: 'primary',
+                        ariaLabel: 'Confirmation text',
+                    },
+                } as ModalProps,
+                slots: {
+                    default: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Deleniti fugit id exercitationem repellendus in magni
```

Comment:
We might want to look at generating this. I know there are tools like faker or lorem ipsum that allow you to pull in content via a simple object so we wont have lots of fake data in our tests.